server/index.js

```diff
@@ -1,9 +1,16 @@
 const express = require("express");
 require("dotenv").config();
 const http = require("http");
+const EventEmitter = require("events");
 const { Server } = require("socket.io");
 const cors = require("cors");
-const route = require("./route");
+const route = require("./routes/route");
+const { addUser } = require("./users/users");
+
+class MyEmitter extends EventEmitter {}
+const myEmitter = new MyEmitter();
+// increase the limit
+myEmitter.setMaxListeners(25);
```

Comment:
For what this emitter. Didn't get it. Could you explain?

Reply:
I had the following warning on the server when several clients connected to the room:

(node:7204) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 disconnect listeners added to
 [Namespace]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)

I was looking for a solution to the problem in Google. You saw him. Commented and now it doesn't issue a warning.
In the next branch I will remove myEmitter...

Reply:
Ok. Actually I think it is not correct solution for your problem. Can you share a link where you get it.
MaxListenersExceededWarning: Possible EventEmitter memory leak detected. - is a very serious problem. 
Let's meet tomorrow and try to solve it.